src/olympia/conf/dev/settings.py

```diff
@@ -17,11 +17,12 @@
 API_THROTTLING = True
 
 DOMAIN = env('DOMAIN', default='addons-dev.allizom.org')
+SERVICES_DOMAIN = env('SERVICES_DOMAIN', default='services.addons-dev.allizom.org')
 SERVER_EMAIL = 'zdev@addons.mozilla.org'
 SITE_URL = 'https://' + DOMAIN
 INTERNAL_SITE_URL = env('INTERNAL_SITE_URL', default='https://addons-dev.allizom.org')
 EXTERNAL_SITE_URL = env('EXTERNAL_SITE_URL', default='https://addons-dev.allizom.org')
-SERVICES_URL = env('SERVICES_URL', default='https://services.addons-dev.allizom.org')
```

Comment:
`SERVICES_URL` environment variable is no longer used as part of this but it was redundant anyway. Still, to keep things consistent, I've introduced `SERVICES_DOMAIN` env variable in case we need that escape hatch.